modules/ray_tracing/test/tests/outputs/ray_tracing_mesh_output/tests

```diff
@@ -93,6 +93,8 @@
     max_parallel = 4
     requirement = 'The system shall support the mesh output of traced rays into the nemesis format for a transient simulation'
     design = 'RayTracingNemesis.md RayTracingMeshOutput.md'
+    # cannot do recover because we don't have a single exodus file, the first two timesteps won't exist
+    recover = false
```

Comment:
This is the only thing that bothers me.  Is there no workaround?  We should know what time step we're on after a recover, shouldn't we?  Can we not just advance directly to that in the file naming?

Reply:
Not easily, no. That sounds like some complicated logic. Fortunately, recover testing works for the exodus case. It's mostly the same back end.